docs/pki/packetfence.asciidoc

```diff
@@ -3,352 +3,81 @@ ifdef::env-github[]
 :imagesdir: ../images
 endif::[]
 
-This section has been created to give a quick start to install the PacketFence PKI in PacketFence. This guide does not include advanced troubleshooting of EAP-TLS connections. Refer to the relevant documentation of EAP-TLS, RADIUS and OpenSSL for advanced features.
+This section has been created to give a quick start to install the PacketFence PKI in PacketFence. This guide does not include advanced troubleshooting of EAP-TLS connections. Refer to the relevant documentation of EAP-TLS, RADIUS and OpenSSL for advanced features. The PacketFence PKI comes installed by default since PacketFence version 10. All certificates would be saved in the database. If you want to migrate your certificate from the old PacketFence PKI please see the upgrade section.
```

Comment:
This section has been created to give a quick start to configure the PKI in PacketFence